### ISSUE TITLE
how to run "custom_strategy"

### ISSUE THREAD

**Issue by Gregkoi** [BitMEX/sample-market-maker#85]
I install bitmex-market-maker, and get settings updated with key, It works!!!
but when I am trying to implement my own custom trading strategies, and add some strategy code into custom_strategy.py, when I run python custom_strategy.py, nothing happened, no error information, no order has been placed in Testnet Bitmex. Can anyone help to me with this? did I do something wrong?

$ python custom_strategy.py
$ #nothing show

my custom_strategy.py is quite simple: 

import sys
from market_maker.market_maker import OrderManager
class CustomOrderManager(OrderManager):
        buy_orders = []
        sell_orders = []
        ticker = self.exchange.get_ticker()
        buy_orders.append({'price': ticker["buy"], 'orderQty': 100, 'side': "Buy"})
        sell_orders.append({'price': ticker["buy"]+50, 'orderQty': 100, 'side': "Sell"})
        self.converge_orders(buy_orders, sell_orders)
def run() -> None:
    order_manager = CustomOrderManager()
    try:
        order_manager.run_loop()
    except (KeyboardInterrupt, SystemExit):
        sys.exit()

**Comment by Gregkoi** [comment]
custom_strategy.py only define run(), but didn't use it, so I simply add one code Run() to get this script worked.

**Comment by discountry** [comment]
```
self.start_position_buy = ticker["buy"] + self.instrument['tickSize']
AttributeError: 'CustomOrderManager' object has no attribute 'instrument'
```

I've got error log like this, do you have any idea about that?

**Comment by discountry** [comment]
**problem solved.**

This package installed from pip has bugs.

I copied the files from github repo and it works fine now.

**Comment by audiroxx** [comment]
were you using python 2.7?

**Comment by discountry** [comment]
python3.6

**Comment by ftenab** [comment]
Hi, can you share your final custom_strategy.py, Im trying to test but facing difficulties...package installed from pip has bugs??...can you explain how you copied from repo??..link??..

Many thanks..

**Comment by discountry** [comment]
@ftenab just download this github repo and copy to your pip package path.

The pip production version has bugs.

**Comment by audiroxx** [comment]
@Gregkoi can u share where u added the run() to run the custom strategy

**Comment by audiroxx** [comment]
Check this #94 
Problems pretty much explained. Cant find a reason to not close this. Open if u have more questions after following the solution

**Comment by ryanfox** [comment]
Should be resolved in v1.4.
